packages/sampler/stories/default/Routable.js

```diff
@@ -0,0 +1,78 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import React from 'react';
+import Heading from '@enact/ui/Heading';
+import {Linkable, Routable, Route} from '@enact/ui/Routable';
+import {storiesOf} from '@storybook/react';
+
+const LinkedButton = Linkable('button');
+const viewStyle = {padding: '6px', border: '1px solid black'};
```

Comment:
Why don't we apply `scaleToRem` instead of `px`?
For `px`, it might be better to use `1apx`.

Reply:
![image](https://user-images.githubusercontent.com/4239873/101322048-b0192b00-38a9-11eb-8fbb-9668a5e64979.png)
The padding left part is out of the View.

Reply:
Just vecause of my laziness.
I'll fix them.

Reply:
It is false alarm. Please ignore my comment.